### PR TITLE
Add links for AutoVizuA11y

### PR DESCRIPTION
**Pull request by DiogoRDuarte**
Hi! Our DataViz team @Feedzai developed [AutoVizuA11y](https://github.com/feedzai/AutoVizuA11y), a React library that automates the process of creating accessible data visualizations for screen reader users. 
Given its context, we believe that it could be added to this list of resources. What do you think?

### CODE REVIEW

**Comment by frankelavsky** [comment]
Hi Diogo, it's great to see this PR come in. Would you please clarify the language about what the library does? "Accessible visualizations" just isn't a clear enough description.

I might suggest something like: 

```
### AutoVizuA11y

* React library that automates the process of creating keyboard navigation, alternative text, and keyboard shortcuts for data visualizations.
* [AutoVizuA11y Github repo](https://github.com/feedzai/AutoVizuA11y)
* [AutoVizuA11y examples site](https://diogorduarte.github.io/autovizua11y-examples/)
```

**Comment by frankelavsky** [comment]
(This is also a reminder to myself to do an update + overhaul of this doc! It's overdue...)

**Comment by DiogoRDuarte** [comment]
@frankelavsky Description changed based on your suggestion!

File: README.md [hunk 1]
```diff
@@ -261,3 +267,7 @@ Ref: Heer, Jeffrey (2019). Visualization is not Enough.
 ### [Visualizing for the Non‐Visual: Enabling the Visually Impaired to Use Visualization [PDF]](http://users.umiacs.umd.edu/~elm/projects/vis4nonvisual/vis4nonvisual.pdf)
 
 Ref: Choi, J., Jung, S., Park, D. G., Choo, J., & Elmqvist, N. (2019, June). Visualizing for the Non‐Visual: Enabling the Visually Impaired to Use Visualization. In *Computer Graphics Forum* (Vol. 38, No. 3, pp. 249-260).
+
+### [AutoVizuA11y: A Tool to Automate Screen Reader Accessibility in Charts [PDF]](https://onlinelibrary.wiley.com/doi/10.1111/cgf.15099)
```
Comment (frankelavsky) [inline README.md:271]: This isn't technically a link to the PDF, perhaps either use the epdf link from Eurographics (https://onlinelibrary.wiley.com/doi/epdf/10.1111/cgf.15099) or just remove the `[PDF]` tag here.

Comment (frankelavsky) [reply README.md:271]: Otherwise, this is looking great! Ready to merge after this and thanks again for the PR.

Comment (DiogoRDuarte) [reply README.md]: Just changed the link as suggested :)

**Pull request merged**